include/slang/binding/Expression.h

```diff
@@ -140,7 +140,8 @@ class Expression {
 
     /// Binds the left hand side of an assignment-like expression from the given syntax nodes.
     static const Expression& bindLValue(const ExpressionSyntax& lhs, const Type& rhs,
-                                        SourceLocation location, const BindContext& context);
+                                        SourceLocation location, const BindContext& context,
+                                        bool isInout = false);
```

Comment:
Can you make this not defaulted? Anyone who calls this function should provide a value for whether it's an inout or not.

Reply:
Yes, I will make the change.